### DISCUSSION TITLE
TBT value in navigate vs timespan

### DISCUSSION THREAD

**Discussion by aadisarma7** [GoogleChrome/lighthouse#15788] [Q&A] [answered]
I am running a user flow where I first navigate to a page, open a timespan, perform a click, and then scroll to the bottom of the page and close the timespan. I am getting two different values for TBT from navigate and timespan and am trying to understand the difference between these values. Is the TBT from the timespan just the blocking time before the selector I am trying to click on is interactable?

**Comment by adamraine** [comment]
TBT in navigation mode is the blocking time between FCP & TTI.

TBT in timespan mode is the blocking time between the start of the timespan and the end of the timespan.

FWIW it sounds like each mode is testing different things during the page lifecycle so it would make sense for the TBT values to be different.

**Reply by aadisarma7** [reply]
Does the TBT from timespan in my case measure the time it blocks after the click interaction takes place before another interaction can occur? Also, do the thresholds defined for TBT in the documentation apply to the timespan case as well or do these have different thresholds?

**Reply by adamraine** [reply, marked as answer]
In timespan mode the TBT window is bounded by the timespan's start and end times that you define when you run timespan mode. It is not bounded by any user interaction and does not obey the same definition as navigation mode outlined in the documentation.
